integrations/maya/src/main/java/org/dash/wallet/integrations/maya/model/SwapTradeResponse.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.dash.wallet.integrations.maya.model
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class SwapTradeResponse(
+    @SerializedName("data")
+    val `data`: SwapTradeResponseData? = null
+) : Parcelable {
+    companion object {
+        val EMPTY_SWAP_TRADE = SwapTradeUIModel("", "", "", "", "")
+    }
+}
+
+@Parcelize
+data class SwapTradeResponseData(
+    val fee: SwapApiAmount? = null,
+    val id: String? = null,
+    @SerializedName("applied_subscription_benefit")
+    val applied_subscription_benefit: Boolean? = null,
```

Comment:
This class and many others were copied from Coinbase.  We will delete them or modify them in the next stories.